packages/components/src/external-link/index.jsx

```diff
@@ -13,6 +13,7 @@ class ExternalLink extends Component {
 		iconSize: 18,
 		showIconFirst: false,
 		iconComponent: null,
+		localizeUrl: true,
```

Comment:
Won't someone convert this one to a functional component finally? 😅

Reply:
Yeah, I assume everyone just scared to do a regression after :)